### PR TITLE
Add feature private messages

### PR DESCRIPTION
**Pull request by danielsp45**
This pull request allows the bot to send a private message to the promoted Ninja in a embed message with the respective color of the belt that the Ninja earned.

### Achievements
- [x] Send private message to promoted ninja (solves : #15 )

<details>
gongo_screen_shot

### CODE REVIEW

**Review by nelsonmestevao — COMMENTED** [review]
I think it looks really cool ⚪️🟡🔵🟢🟠🔴🟣⚫️

**Comment by nelsonmestevao** [comment]
What do you guys think about the bot message start with one of these emojis: ⚪️🟡🔵🟢🟠🔴🟣⚫️

> "⚫️ Parabéns, agora és cinturão negro 🎉"

@danielsp45 @RuiL1904

File: bot/cogs/belts.py [hunk 1]
```diff
@@ -1,8 +1,31 @@
...
+    
+    def get_info(self):
+        with open(self.file) as json_file:
+                data = json.load(json_file)
+                msg = f"Subiste para {self.belt} :clap:"
+                color = int(data[self.belt]["color"], 16)
+                for param in data[self.belt]["objectives"]:
+                    msg += '\n' + param
+
+        return (
```
[the remark this answers is not in the document]
Comment (danielsp45) [reply bot/cogs/belts.py:24]: You're right I'll change it.

**Comment by danielsp45** [comment]
> What do you guys think about the bot message start with one of these emojis: ⚪️🟡🔵🟢🟠🔴🟣⚫️
> 
> > "⚫️ Parabéns, agora és cinturão negro 🎉"
> 
> @danielsp45 @RuiL1904

I think it looks cool. Should I put it in?

**Comment by RuiL1904** [comment]
> > What do you guys think about the bot message start with one of these emojis: white_circleyellow_circlelarge_blue_circlegreen_circleorange_circlered_circlepurple_circleblack_circle
> > > "black_circle Parabéns, agora és cinturão negro tada"
> > 
> > 
> > @danielsp45 @RuiL1904
> 
> I think it looks cool. Should I put it in?

Seems a good idea.

File: bot/cogs/belts.json [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+            "- Saber o nome de 7 ninjas e 4 mentores",
+            "- Conseguir completar o código que os mentores têm preparado para ti",
+            "- Fazer um site"],
+        "color" : "296DF7"
+        },
+    "Verde" : {
+        "objectives" : [
+            "- Apresentar um projeto para o Dojo inteiro",
+            "- Chegar a 5 Kyu em CodeWars",
+            "- Missão secreta nº2",
```
Comment (nelsonmestevao) [inline bot/cogs/belts.json:28]:
```suggestion
            "- Missão secreta n.º2",
```

Since "n" is an abbreviation of "número" it must have a dot.

Comment (danielsp45) [reply bot/cogs/belts.json]: I changed it in the most recent commit and also added the circle emoji to the private message as requested.

**Review by RuiL1904 — APPROVED** [review]
Nice work. Keep up! 🙌🏼

**Pull request merged**